plugins/repository-s3/src/main/java/org/opensearch/repositories/s3/S3BlobContainer.java

```diff
@@ -296,6 +297,23 @@ private static DeleteObjectsRequest bulkDelete(String bucket, List<String> blobs
             .build();
     }
 
+    @Override
+    public void listBlobsByPrefixInLexicographicOrder(String blobNamePrefix, int limit, ActionListener<List<BlobMetadata>> listener) {
```

Comment:
Will change the name as per comment: https://github.com/opensearch-project/OpenSearch/pull/8314#discussion_r1245563313

Reply:
Addressed as part of https://github.com/opensearch-project/OpenSearch/pull/8314/commits/e85d14f7c520ff883ae0faca243040d2cfc8074f